### ISSUE TITLE
Local variable and member variable have the same name

### ISSUE THREAD

**Issue by levi106** [Microsoft/intelligent-apps#83]
## Project

ConsotoHelpdeskChatBot

## File

InstallAppDialog.cs

## Description

InstallAppDialog class has a member named "install". And also, multipleAppsStepAsync method has a local variable with the same name. I think it's a bad practice because it may cause an unexpected behavior.

**Comment by leo-leong** [comment]
Agreed. We are open to changes and feel free to send a PR in if there's a better way to do this.
